source/partials/ic-upstream-structure.md

```diff
@@ -30,7 +30,7 @@ code/
     - `composer.json`: Composer automatically updates `composer.json` with customizations for the upstream. Avoid manually modifying this file.
 - `pantheon.upstream.yml`: The `build_step: true` directive in `pantheon.upstream.yml` enables the build step.
 
-When a site is created, Pantheon runs `composer install`, generates a `composer.lock` file, and commits it back to the site’s code repository. To avoid potential merge conflicts after applying an upstream update, do not commit the `composer.lock` file to the upstream repository.
+When a site is created, Pantheon runs `composer install`, generates a `composer.lock` file, and commits it back to the site’s code repository. To avoid potential merge conflicts after applying an upstream update, do not commit the `composer.lock` file from your upstream root to the upstream repository.
```

Comment:
We could maybe add a little helpful advice for working with custom upstreams locally:
```
Since the `composer.lock` file should be committed for individual sites created from your custom upstream, it should not be listed in your .gitignore file. To make it easier to avoid accidentally commit a `composer.lock` file when working directly with your custom upstream, run the following command:

    echo /composer.lock >> .git/info/exclude

This will have the same effect as ignoring the file in the top-level .gitignore file, except that the `exclude` file is not saved in the repository when you push changes back to the remote origin. You therefore must re-run this command every time you make a new local clone of your upstream if you would like to protect against inadvertently committing a composer.lock file.
```

Reply:
It isn't necessary to ever make a composer.lock file in the custom upstream, but it might happen if folks test `composer update` in their upstream. I am therefore somewhat undecided about how important it is to include this advice. It's helpful, so I am inclined to include it.